CommonConfig.mjs

```diff
@@ -0,0 +1,447 @@
+        "no-const-assign": ["error"],
+        "no-constant-binary-expression": ["error"],
+        "no-constant-condition": [
+            "error",
+            {checkLoops: false},
+        ],
+        "no-constructor-return": ["error"],
+        "no-continue": ["off"],
+        "no-control-regex": ["error"],
+        "no-debugger": ["error"],
+        "no-delete-var": ["error"],
+        "no-div-regex": ["error"],
+        "no-dupe-args": ["error"],
+        "no-dupe-class-members": ["error"],
+        "no-dupe-else-if": ["error"],
+        "no-dupe-keys": ["error"],
+        "no-duplicate-case": ["error"],
+        "no-duplicate-imports": ["error"],
+        "no-else-return": ["error"],
+        "no-empty": ["error"],
+        "no-empty-character-class": ["error"],
+        "no-empty-function": [
+            "error",
+            {allow: ["arrowFunctions"]},
+        ],
+        "no-empty-pattern": ["error"],
+        "no-empty-static-block": ["error"],
+        "no-eq-null": ["error"],
+        "no-eval": ["error"],
+        "no-ex-assign": ["error"],
+        "no-extend-native": ["error"],
+        "no-extra-bind": ["error"],
+        "no-extra-boolean-cast": ["error"],
+        "no-extra-label": ["error"],
+        "no-fallthrough": ["error"],
+        "no-func-assign": ["error"],
+        "no-global-assign": ["error"],
+        "no-implicit-coercion": ["error"],
+        "no-implicit-globals": ["error"],
+        "no-implied-eval": ["error"],
+        "no-import-assign": ["error"],
+        "no-inline-comments": ["error"],
+        "no-inner-declarations": ["error"],
+        "no-invalid-regexp": ["error"],
+        "no-invalid-this": ["off"],
+        "no-irregular-whitespace": ["error"],
+        "no-iterator": ["error"],
+        "no-label-var": ["error"],
+        "no-labels": ["error"],
+        "no-lone-blocks": ["error"],
+        "no-lonely-if": ["error"],
+        "no-loop-func": ["error"],
+        "no-loss-of-precision": ["error"],
+        "no-magic-numbers": [
+            "error",
+            {
+                detectObjects: false,
+                enforceConst: true,
+                ignore: [
+                    -1,
+                    0,
+                    1,
+                    2,
+                    3,
+                    10,
+                    100,
+                ],
+                ignoreArrayIndexes: true,
+                ignoreClassFieldInitialValues: true,
+                ignoreDefaultValues: true,
+            },
+        ],
+        "no-misleading-character-class": ["error"],
+        "no-multi-assign": ["error"],
+        "no-multi-str": ["error"],
+        "no-negated-condition": ["off"],
+        "no-nested-ternary": ["error"],
+        "no-new": ["error"],
+        "no-new-func": ["error"],
+        "no-new-native-nonconstructor": ["error"],
+
+        // deprecated in ESLint v9.0.0 and replaced by the no-new-native-nonconstructor
+        "no-new-symbol": ["error"],
```

Comment:
If this is deprecated, why are we keeping it?

Reply:
disabled now

Reply:
Hm, can we remove it entirely or is there a benefit to keeping it?

Reply:
I think it's better to keep the deprecated rules so that we know those has been configured / looked at.